doc/classes/AABB.xml

```diff
@@ -33,211 +33,314 @@
 			<param index="0" name="position" type="Vector3" />
 			<param index="1" name="size" type="Vector3" />
 			<description>
-				Constructs an [AABB] from a position and size.
+				Constructs an [AABB] by [param position] and [param size].
 			</description>
 		</constructor>
 	</constructors>
 	<methods>
 		<method name="abs" qualifiers="const">
 			<return type="AABB" />
 			<description>
-				Returns an AABB with equivalent position and size, modified so that the most-negative corner is the origin and the size is positive.
+				Returns an [AABB] equivalent to this bounding box, with its width, height, and depth modified to be non-negative values.
+				[codeblocks]
+				[gdscript]
+				var box = AABB(Vector3(5, 0, 5), Vector3(-20, -10, -5))
+				var absolute = box.abs()
+				print(absolute.position) # Prints (-15, -10, 0)
+				print(absolute.size)     # Prints (20, 10, 5)
+				[/gdscript]
+				[csharp]
+				var box = new AABB(new Vector3(5, 0, 5), new Vector3(-20, -10, -5))
+				var absolute = box.Abs();
+				GD.Print(absolute.position); // Prints (-15, -10, 0)
+				GD.Print(absolute.size);     // Prints (20, 10, 5)
+				[/csharp]
+				[/codeblocks]
+				[b]Note:[/b] It's recommended to use this method when [member size] is negative, as most other methods in Godot assume that the [member size]'s components are greater than [code]0[/code].
 			</description>
 		</method>
 		<method name="encloses" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="with" type="AABB" />
 			<description>
-				Returns [code]true[/code] if this [AABB] completely encloses another one.
+				Returns [code]true[/code] if this bounding box [i]completely[/i] encloses the [param with] box.
```

Comment:
Id suggest adding details on what this exactly means, e.g. if `AABB(0, 0, 10, 10)` encloses `AABB(0, 0, 5, 5)` or only `AABB(1, 1, 5, 5)`.

Reply:
Testing I found this quirk:
```gdscript
var a := AABB(Vector3(0, 0, 0), Vector3(10, 10, 10))
var b := AABB(Vector3(0, 0, 0), Vector3(5, 5, 5))
var c := AABB(Vector3(5, 5, 5), Vector3(5, 5, 5))
print(a.encloses(b)) # Prints "true"
print(a.encloses(c)) # Prints "false"!
```
This quirk (might be a bug) should be clarified IMO

Reply:
Explaining what "enclose" means is more trouble than it is worth because it is much, much easier to imagine it. Perhaps an extra "Edges are included" and a few examples to show the quirks may do the trick.

Reply:
Examples would fix that, because imagining it is error prone for what it truly means, will look into that bug as it was fixed for Rect2 and Rect2i, will get back to you with the results (and if this should be updated or not)

Reply:
When that PR is merged, how will that affect the current example?

```GDScript
var a = AABB(Vector3(0, 0, 0), Vector3(4, 4, 4))
var b = AABB(Vector3(0, 0, 0), Vector3(2, 2, 2))
var c = AABB(Vector3(2, 2, 2), Vector3(2, 2, 2))

print(a.encloses(a)) # Prints false
print(a.encloses(b)) # Prints true
print(a.encloses(c)) # Prints false
```

Would all of them print `true`?

Reply:
They will!

Reply:
I see. They would need to be changed to have one "false" example, then.

Reply:
Note also that this is marked for 4.1 but my PR is (currently) not marked for 4.1 so the documentation should not be applied for 4.1 with this detail

Reply:
And merged